packages/amplify-codegen-appsync-model-plugin/src/utils/process-auth.ts

```diff
@@ -26,7 +26,7 @@ export enum AuthModelMutation {
 }
 
 const DEFAULT_GROUP_CLAIM = 'cognito:groups';
-const DEFAULT_USER_CLAIM = 'username';
+const DEFAULT_IDENTITY_CLAIM = 'username';
```

Comment:
Shouldn't this be "cognito:userName"?

Reply:
If you mean the case sensitive, its should be `cognito:username`
https://github.com/aws-amplify/amplify-cli/blob/830ec21adbdd6365318a983aa8075b7b8802a444/packages/graphql-auth-transformer/src/resources.ts#L42

If you mean it we should default to `cognito:username` then it happens when we set the value here on line 65
```ts
identityClaim: identityClaim === 'username' ? 'cognito:username' : identityClaim,
```

Reply:
Ah, got it.